src/Three.js

```diff
@@ -156,5 +156,8 @@ export { Curve } from './extras/core/Curve.js';
 export { ShapeUtils } from './extras/ShapeUtils.js';
 export { SceneUtils } from './extras/SceneUtils.js';
 export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
+export { WEBGL_CONSTANTS } from './renderers/webgl/WebGLConstants.js';
```

Comment:
@mrdoob I've updated `WebGLUtils` and `WebGLConstants` (still WIP) but to show a possible implementation of this last option